test/infra/infra-dbdeployer-mysql84/docker/entrypoint.sh

```
@@ -0,0 +1,149 @@
+done
+
+# Create test users on all 3 nodes
+# ROOT_PASSWORD and INFRA are passed via environment variables
+ROOT_PASSWORD="${ROOT_PASSWORD:-default_password}"
+INFRA="${INFRA:-infra-dbdeployer-mysql84}"
+
+echo "Creating test users on all nodes..."
+for PORT in "${NODE_PORTS[@]}"; do
+    echo "Configuring users on port ${PORT}..."
+
+    ${MYSQL_CMD} -P${PORT} <<SQL
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)